### ISSUE TITLE
Colima can't connect to docker socket

### ISSUE THREAD

**Issue by philipphofmann** [getsentry/devenv#95]
When running `devenv bootstrap` or `devenv sync`, Colima waits for docker to start but can't connect to docker. Trying to run any docker command fails with something like
```
Can't connect to docker socket: socket: unix:///Users/philipp.hofmann/.colima/default/docker.sock
```

Checking the Colima deamon.log under `~.colima/default/daemon.log` shows the following error
```
time="2024-02-28T10:33:05+01:00" level=error msg="error fetching docker volumes: error listing containers: error running [lima docker ps -q], output: \"permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get \\\"http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json\\\": dial unix /var/run/docker.sock: connect: permission denied\", err: \"exit status 1\"" context=inotify
```

There is a related issue with Colima, which was fixed with https://github.com/abiosoft/colima/issues/878 and which is in the latest Colima version we use currently, `v0.6.8`. The fix ensures that the user is added to the docker group, which made me think if running `devenv bootstrap` or `devenv sync` doesn't have some proper permissions to add the user to the docker group when installing Colima. So, I removed the sentry-devenv by deleting the folder from `.local/share/,` installed Colima via homebrew, ensured that Colima could talk to docker with `colima start` and `docker ps` and didn't get any error anymore. Then, I installed the devenv again, and `devenv bootstrap` worked and Colima could talk to docker.

It took me a while to figure this out, so I opened this issue. I do not know the problem, but I wanted to share it with you so you can maybe work on a fix.

**Comment by joshuarli** [comment]
Yeah, this is part of the reason we're upgrading to colima 0.6.8! Coming very soon.

**Comment by joshuarli** [comment]
Doing some housekeeping - this was fixed a long time ago with the upgrade to 0.6.6. That fix was shipped as part of https://github.com/abiosoft/colima/releases/tag/v0.6.4.
